aio/aio-proxy/aio_proxy/doc/open-api.yml

```diff
@@ -792,6 +792,12 @@ paths:
                           type: string
                           example: "H"
                           description: Calculée à partir de l'activté principale.
+                        slug:
```

Comment:
I wonder : should we call it `slug` ? Or should we choose a varaible name more closely related to annuaire ? 
Like `annuaire-entreprises-path`